addons/project/views/project_views.xml

```diff
@@ -1114,16 +1127,23 @@
                                     <field name="legend_blocked" invisible="1"/>
                                     <field name="project_id" invisible="1"/>
                                     <field name="is_closed" invisible="1"/>
-                                    <field name="sequence" widget="handle"/>
-                                    <field name="name"/>
+                                    <field name="allow_subtasks" invisible="1" />
+                                    <field name="priority" widget="priority" optional="show" nolabel="1"/>
+                                    <field name="id" optional="hide"/>
+                                    <field name="name" widget="name_with_subtask_count"/>
                                     <field name="display_project_id" string="Project" optional="hide"/>
+                                    <field name="child_text" invisible="1"/>
```

Comment:
I would place it just before the `name` field as we need it because of `name_with_subtask_count`